apps/crn-server/src/data-providers/contentful/user.data-provider.ts

```diff
@@ -217,7 +217,7 @@ const cleanUser = (userToUpdate: UserUpdateDataObject) =>
         connections: connections.map(({ code }) => code),
       };
     }
-    return { ...acc, [key]: value };
+    return { ...acc, [key]: value === '' ? null : value };
```

Comment:
do you think maybe we could leave as `value.trim() === ''`? not sure if the user could leave with empty space by mistake in this flow

Reply:
For this particular issue - the degree field in onboarding - they cannot because it's a drop-down menu, but generally it's probably worth it.